bin/dkr-build-test-container

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -e
+
+SCRIPT_DIR=$( dirname "$0" )
+source "$SCRIPT_DIR/dkr-config"
+
+echo docker image rm "${CONTAINER_NAME}:${CONTAINER_VERSION}" || true 
```

Comment:
It's not the Container name/version, but the image.

Reply:
changed